src/prog_models/models/battery_circuit.py

```diff
@@ -20,7 +20,10 @@ class BatteryCircuit(prognostics_model.PrognosticsModel):
         i: Current draw on the battery
 
     States: (4)
-        tb, qb, qcp, qcs
+        | tb : Battery Temperature (°C)
+        | qb : Battery Charge
+        | qcp : 
```

Comment:
qb : charge stored in Capacitor Cb of the equivalent circuit model
qcp : charge stored in Capacitor Ccp of the equivalent circuit model
qcs : charge stored in Capacitor Ccs of the equivalent circuit model